Dockerfile

```diff
@@ -9,11 +9,26 @@ RUN corepack prepare pnpm@latest --activate
 RUN pnpm install
 RUN pnpm build
 
-FROM caddy:alpine
+FROM golang:1.20-bullseye as build-bundle
```

Comment:
We may consider using `debian-12` as base image

Reply:
I simply copy / pasted from dae

Reply:
Let's leave it as it is for now.

Reply:
OK